### PR TITLE
Translated #17. Ready for proofreading.

### PR DESCRIPTION
**Pull request by lruzicka**
This is a translation of the above mentioned section, ready for proofreading.

### CODE REVIEW

File: docs/handout_en.md [hunk 1]
```diff
@@ -213,22 +213,24 @@ Na pracovní plochu se přesunete již zmíněnou zkratkou CTRL+ALT+D. Další
...
+shortcut. When you have switched to the panel, use the **Tab** and
+**Shift+Tab** to thumb trough the particular items. On the panel, you 
+will find icons for network connections, volume settings, battery
+indicator, and further notifications.
+
+Unfortunately, the icons may not be correctly announced, especially after
+the system has just started. It is recommended to navigate throughout the
+whole panel using the **Tab** key first to have the icons announced
+correctly on consecutive passes. You can activate the icon by pressing
+the **Space** key.
```
Comment (vojtapolasek) [inline docs/handout_en.md:229]:
```suggestion
the **Spacebar** key.
```

File: docs/handout_en.md [hunk 2]
```diff
@@ -213,22 +213,24 @@ Na pracovní plochu se přesunete již zmíněnou zkratkou CTRL+ALT+D. Další
...
+
+You can add more panels, or add icons to the current panels. Use
```
Comment (vojtapolasek) [inline docs/handout_en.md:231]:
```suggestion
You can add more panels, or add icons to already existing panels. Use 
```

File: docs/handout_en.md [hunk 3]
```diff
@@ -213,22 +213,24 @@ Na pracovní plochu se přesunete již zmíněnou zkratkou CTRL+ALT+D. Další
...
+indicator, and further notifications.
+
+Unfortunately, the icons may not be correctly announced, especially after
+the system has just started. It is recommended to navigate throughout the
+whole panel using the **Tab** key first to have the icons announced
+correctly on consecutive passes. You can activate the icon by pressing
+the **Space** key.
+
+You can add more panels, or add icons to the current panels. Use 
+the **Shift+F10** combination to open the corresponding dialogue.
```
Comment (vojtapolasek) [inline docs/handout_en.md:232]:
```suggestion
the **Shift+F10** combination to open the corresponding menu.
```

**Review by vojtapolasek — CHANGES_REQUESTED** [review]
Thank you, see comments please.

File: docs/handout_en.md [hunk 4]
```diff
@@ -213,22 +213,24 @@ Na pracovní plochu se přesunete již zmíněnou zkratkou CTRL+ALT+D. Další
...
-
-je možné přidat další panely, či na stávající panely přidat další ikony.
-Příslušnou nabídku otevřete klávesovou zkratkou Shift+F10. Na pracovní
-plochu se můžete opět vrátit zkratkou CTRL+ALT+D.
+## Panels
+
+By default, there is one panel placed above the workspace. You can switch
+between the workspace and the panels using the **Ctrl+Alt+Tab** keyboard
+shortcut. When you have switched to the panel, use the **Tab** and
+**Shift+Tab** to thumb trough the particular items. On the panel, you
```
[the remark this answers is not in the document]
Comment (lruzicka) [reply docs/handout_en.md]: Changed.

**Comment by lruzicka** [comment]
I have made the required changes.

**Review by vojtapolasek — APPROVED** [review]

**Comment by vojtapolasek** [comment]
Hello, thank you very much for the changes. Merging.

**Pull request merged**